src/lib/chips/chips.md

```diff
@@ -22,13 +22,31 @@ _Hint: `<md-basic-chip>` receives the `mat-basic-chip` CSS class in addition to
 Chips can be selected via the `selected` property. Selection can be disabled by setting
 `selectable` to `false` on the `<md-chip-list>`.
 
-Selection emits the `(select)` output while deselecting emits the `(deselect)` output. Both outputs
-receive a ChipEvent object with a structure of `{ chip: alteredChip }`.
+Whenever the selection state changes, a `ChipSelectionChange` event will be emitted via 
+`(onSelectionChange)`.
 
 ### Disabled chips
 Individual chips may be disabled by applying the `disabled` attribute to the chip. When disabled,
 chips are neither selectable nor focusable. Currently, disabled chips receive no special styling.
 
+### Chip input
+The `MdChipInput` directive can be used together with a chip-list to streamline the interaction
+between the two components. This directive adds chip-specific behaviors to the input element
+within `<md-form-field>` for adding and removing chips. The `<input>` with `MdChipInput` can
+be placed inside or outside the chip-list element.
+
+<!-- example(chips-input) -->
+
+```html
```

Comment:
Add a title to this example to clarify what it's show that the live example isn't?

Reply:
Added description. PTAL. Thanks!